### ISSUE TITLE
Missing details about training

### ISSUE THREAD

**Issue by JoakimEdin** [3mcloud/MDACE#5]
I have two questions about the training of the supervised attention method:

1. Do you first train EffectiveCan on the MIMIC-III full dataset and then afterward fine-tune the supervised attention, or do you use the combined dataset and train both simultaneously?
2. Which value of lambda did you use in the loss function?

**Comment by ranajafari** [comment]
We trained the supervised attention simultaneously. I.e., the training dataset consists of unannotated and annotated encounters (which we refer to as Code + Evidence in Table 8), and attention is supervised only for the codes with annotations.

For the weight values, λ₁ = 2.5 was used for supervised attention, and for tagging λ₂ = 0.5. The values were determined such that the overall code prediction performance of the ECAN model would not be degraded. Appendix D discusses some information on model parameters.
